src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,28 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.dao.TicketDao;
```

Comment:
Consider removing the direct usage of DAOs (`OrderDao` and `TicketDao`) and replace them with their respective services to adhere to a service-oriented architecture. This will ensure better encapsulation and abstraction.